recipes/muparser/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set version = "2.2.5" %}
+
+package:
+  name: muparser
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/beltoforion/muparser/archive/v{{ version }}.tar.gz 
+  md5: 02dae671aa5ad955fdcbcd3fee313fb7
+
+build:
+  number: 0
+  skip: true  # [win]
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libmuparser.so  # [linux]
+    - test -f ${PREFIX}/lib/libmuparser.dylib  # [osx]
+
+about:
+  home: http://muparser.beltoforion.de/
+  license: MIT
+  summary: A fast math parser library
+
+extra:
+  recipe-maintainers:
+    - jschueller
+
```

Comment:
I have raised the issue on this point. Still would like to see this simple change.